quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,618 @@
+        QChemDictSet's overwrite_inputs. For example: {"rem": {"symmetry": "true"}}.
+    opt_swaps
+        Dictionary of custom kwargs for run_ase_opt
+            opt_defaults = {"fmax": 0.01, "max_steps": 1000, "optimizer": "Sella"}
+
+    Returns
+    -------
+    dict
+        Dictionary of results from quacc.schemas.ase.summarize_opt_run
+    """
+
+    # Reminders to self:
+    #   - exposing TRICs?
+    #   - passing initial Hessian?
+
+    true_charge, true_spin = true_charge_and_spin(atoms, charge, multiplicity)
+
+    if direction not in ["forward", "reverse"]:
+        raise ValueError("direction must be 'forward' or 'reverse'! Exiting...")
+
+    opt_swaps = opt_swaps or {}
+    opt_defaults = {
+        "fmax": 0.01,
+        "max_steps": 1000,
+        "optimizer": IRC,
+        "optimizer_kwargs": {},
+        "run_kwargs": {"direction": direction},
+    }
+    opt_flags = opt_defaults | opt_swaps
+    if "sella.optimize" not in opt_flags["optimizer"].__module__:
+        raise RuntimeError("Only Sella should be used for TS optimization! Exiting...")
+
+    if pcm_dielectric is not None and smd_solvent is not None:
+        raise ValueError("PCM and SMD cannot be employed simultaneously! Exiting...")
+
+    overwrite_inputs = {"rem": {"method": xc}}
+
+    calc_swaps = calc_swaps or {}
+    for key in calc_swaps:
+        if key not in overwrite_inputs:
+            overwrite_inputs[key] = {}
+        for subkey in calc_swaps[key]:
+            overwrite_inputs[key][subkey] = calc_swaps[key][subkey]
+
+    qchem_input_params = {
+        "basis_set": basis,
+        "scf_algorithm": scf_algorithm,
+        "qchem_version": 6,
+        "pcm_dielectric": pcm_dielectric,
+        "smd_solvent": smd_solvent,
+        "overwrite_inputs": overwrite_inputs,
+    }
+
+    if scf_algorithm.lower() == "gdm":
+        qchem_input_params["max_scf_cycles"] = 200
+
+    calc = QChem(
+        input_atoms=atoms,
+        cores=cores,
+        charge=charge,
+        spin_multiplicity=multiplicity,
+        qchem_input_params=qchem_input_params,
+    )
+    atoms.calc = calc
+    dyn = run_ase_opt(atoms, **opt_flags)
+
+    return summarize_opt_run(
+        dyn,
+        check_convergence=check_convergence,
+        charge_and_multiplicity=(true_charge, true_spin),
+        additional_fields={"name": "Q-Chem IRC Optimization"},
+    )
+
+
+@ct.electron
+@requires(
+    Sella,
+    "Sella must be installed. pip install sella",
+)
+def quasi_irc_job(
+    atoms: Atoms,
+    direction: str,
+    cores: int | None = None,
+    charge: int | None = None,
+    multiplicity: int | None = None,
+    xc: str = "wb97mv",
+    basis: str = "def2-svpd",
+    scf_algorithm: str = "diis",
+    pcm_dielectric: str | None = None,
+    smd_solvent: str | None = None,
+    calc_swaps: dict | None = None,
+    irc_swaps: dict | None = None,
+    relax_swaps: dict | None = None,
+    check_convergence: bool = True,
```

Comment:
This function can/should be greatly cleaned up. Refer to [this example](https://github.com/Quantum-Accelerators/quacc/blob/ccb4de931c5ee46ab7c5f3355c52d98c3a41025b/quacc/recipes/emt/slabs.py). Anytime you have a multi-step "job" like you have here, the better option is to pass the `irc_job` and `relax_job` as function kwargs. Then you can get rid of basically all of your kwargs here.